src/services/csvUpload/load_csv_data.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+var dotenv = require('dotenv');
+dotenv.config();
+
+var extend = require('lodash').assign;
+var mysql = require('mysql');
+
+
+
+function getConnection () {
+  return mysql.createConnection(extend({
+    database: 'STM_DB',
+    host: process.env.MYSQL_HOST,
+    user: process.env.MYSQL_USER,
+    password: process.env.MYSQL_PASSWORD
+  }));
+}
+
+function insert_student (data, cb) {
+    var connection = getConnection();
+    var data2 = data;
+    var data3 = data2.concat(data);
+    console.log(data3);
+    connection.query(
+        'INSERT INTO `student` (id, lastName, firstName, sex, dob) VALUES (?, ?, ?, ?, ?) ON DUPLICATE KEY UPDATE  id = ?, `lastName`= ?, `firstName`=?, `sex`= ?, `dob`=?;',
+        data3,
+        cb
+    );
+    connection.end();
+}
+
+function insert_ydsd(data,cb){
+  var connection = getConnection();
+    var comment = connection.query(
+    'SELECT comments from ydsd where id = ?;',
+    data[0],
+    cb
+    );
+    if(commment!='' && data[2] ==''){
+       data[2] = commment;
+    }
+
+  var data2 = data;
+  var data3 = data2.concat(data);
+  console.log(data3);
+  connection.query(
+      'INSERT INTO `ydsd` (id, year, comments, homeroomTeacher, asp, nextMeetingSch, advancedMath, speechLanguage, studentDevelopment, mathEnrichment, IUreadingServices, IUmathServices, earobics, behavior, workEthic, youngestChild, onlyChild, newStudent, medicalConcern, hmp, dra, RAZ, WTW, iStation, mathBench, Dibels, cogAT, IOWA, elaTotal, ExtendedELA, mathTotal, facultyStudent, potentialDelay, behaviorObservation, selfHelp, socialEmotional, dial4, gradeEntering, ge) VALUES (?,?,?, ?, ?, ?,?, ?, ?, ?, ?,?, ?, ?, ?, ?,?, ?, ?, ?, ?,?, ?, ?, ?, ?,?, ?, ?, ?, ?,?, ?, ?, ?, ?,?,?, ?) ON DUPLICATE KEY UPDATE  `id` = ?, `year`=?, `comments`=?, `homeroomTeacher`=?, `asp`=?, `nextMeetingSch`=?, `advancedMath`=?, `speechLanguage`=?, `studentDevelopment`=?, `mathEnrichment`=?, `IUreadingServices`=?, `IUmathServices`=?, `earobics`=?, `behavior`=?, `workEthic`=?, `youngestChild`=?, `onlyChild`=?, `newStudent`=?, `medicalConcern`=?, `hmp`=?, `dra`=?, `RAZ`=?, `WTW`=?, `iStation`=?, `mathBench`=?, `Dibels`=?, `cogAT`=?, `IOWA`=?, `elaTotal`=?, `ExtendedELA`=?, `mathTotal`=?, `facultyStudent`=?, `potentialDelay`=?, `behaviorObservation`=?, `selfHelp`=?, `socialEmotional`=?, `dial4` =?, `gradeEntering` = ?, `ge`=?;',
+      data3,
+      cb
+    );
+
+  connection.end();
+}
+
+
+module.exports = {
+    insert_student: insert_student,
+    insert_ydsd: insert_ydsd
+};
+
+if (module == require.main) {
```

Comment:
I'm not entirely sure what this does, but I don't think you need to run depending on the module. It'll be run when the service is called from the api